### ISSUE TITLE
The configured rules are not in effect

### ISSUE THREAD

**Issue by p-moon** [dan-lovelace/word-replacer-max#3]
<img width="1577" alt="image" src="https://github.com/dan-lovelace/word-replacer-max/assets/16488309/38160381-daa8-4b19-b035-13fad1d123ed">

**Comment by dan-lovelace** [comment]
Hello @p-moon,

I've been looking into this issue but have not found anything yet. I was able to extract the characters from your screenshot but am not able to reproduce. On my side, the sequence of characters `面`, `试` and `准` are replaced as expected:

|Before|After|
|-|-|
|![Screenshot 2024-04-11 at 10 21 16 AM](https://github.com/dan-lovelace/word-replacer-max/assets/13373818/3818c7a1-cb16-4b13-9bec-5a5c92941ce8)|![Screenshot 2024-04-11 at 10 22 08 AM](https://github.com/dan-lovelace/word-replacer-max/assets/13373818/4d83c5c0-ff10-48ff-9ef6-48528833eb52)|

I wonder if there are hidden characters between the ones you're searching? If that is the case, you might have success with a regular expression instead of case-match. Try the search pattern `面.*试.*准.*` using the Regular Expression option instead like this:

|Regular expression|
|-|
|![Screenshot 2024-04-11 at 10 18 18 AM](https://github.com/dan-lovelace/word-replacer-max/assets/13373818/119b9cb0-c1cc-40af-ae03-60cb82fbe1f5)|

I'll leave the original sequence here if you'd like to test:

面试准

Let me know if you have any luck with the regular expression.

**Comment by dan-lovelace** [comment]
Hi @p-moon - Just checking in to see if you've had any luck with the regular expression pattern provided in my previous comment. Is this still an issue for you?

**Comment by p-moon** [comment]
<img width="1360" alt="image" src="https://github.com/dan-lovelace/word-replacer-max/assets/16488309/bb00d33a-ed11-434b-934c-292565d5c1fa">

I found that the content in the head tag has not been replaced

**Comment by dan-lovelace** [comment]
@p-moon thank you for the idea. I've made an update to include the head's `title` elements which should solve your latest issue (see below). It is part of v0.2.0 which I've submitted to both the Firefox and Chrome stores. Chrome usually takes a day or so to publish new versions but it is coming out soon.

I'll leave this issue open for a bit to make sure you're all set.

![Screenshot 2024-05-16 at 6 06 36 PM](https://github.com/dan-lovelace/word-replacer-max/assets/13373818/bf5ef0d6-26c9-48b4-be70-253677b0cc43)
